saged/impute_expression.py

```diff
@@ -67,7 +61,13 @@
     subset_percents = []
     val_losses = []
     for i in range(len(cv_splits)):
-        for subset_number in range(1, 11):
+        for subset_number in [2, 4, 8]:
```

Comment:
Is there a reason why you don't want to just loop through `subset_percent` here directly, rather than looping through `subset_number` then converting it to `subset_percent` on line 71? It doesn't look like you're using `subset_number` anywhere else, but maybe I'm missing something.

Reply:
Yeah it's an artifact of the old `in range(1, 11)` line here. Range doesn't let me iterate by .1 (or at least I wasn't able to figure out how to force it to), so I stuck with the 1-10 range. I imagine I'll switch back to a linear progression at some point so I'm fine with having this be two statements for now